packages/@aws-cdk/aws-batch-alpha/lib/ecs-container-definition.ts

```diff
@@ -950,7 +965,22 @@ export class EcsFargateContainerDefinition extends EcsContainerDefinitionBase im
     super(scope, id, props);
     this.assignPublicIp = props.assignPublicIp;
     this.fargatePlatformVersion = props.fargatePlatformVersion;
+    this.ephemeralStorageSize = props.ephemeralStorageSize;
     this.executionRole = props.executionRole ?? createExecutionRole(this, 'ExecutionRole');
+
+    // validates ephemeralStorageSize is within limits
+    if (props.ephemeralStorageSize) {
+      this.node.addValidation({
+        validate() {
+          if (props.ephemeralStorageSize!.toGibibytes() > 200) {
+            return ['Ephemeral Storage must be at most 200 GiB.'];
+          } else if (props.ephemeralStorageSize!.toGibibytes() < 21) {
+            return ['Ephemeral Storage must be minimum 21 GiB.'];
```

Comment:
Take a look at how we structure our other error messages and see if you can come up with the best version here.

Reply:
Hello @sumupitchayan 

Do you need some help to complete the PR ?
Looking at the review, this is the only comment that was not fixed.

Looking at `ComputeEnvironments` [here](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-batch-alpha/lib/managed-compute-environment.ts#L1176-L1183) an example of message could be:

> ECS Fargate container '${id}' has 'ephemeralStorage' = ${props.ephemeralStorageSize} > 200 GiB; 'ephemeralStorage' cannot be greater than 200 GiB

Sorry in advance for direct ping, we're looking actively on this fix as it would avoid us stepping back from alpha package ...